liquid_node/commands.py

```diff
@@ -246,14 +246,26 @@ def halt():
 def gc():
     """Stop collections jobs that are no longer declared in the ini file."""
 
-    nomad_jobs = nomad.jobs()
-
-    for job in nomad_jobs:
+    killed_jobs = []
```

Comment:
`jobs_to_stop`?

Reply:
The list will contain jobs that were killed in the loop, not jobs that are to be stopped after the loop.

Reply:
The nomad command is called `stop`, not `kill`. There is no killing or force stopping happening here.

Also, when you declare the list, nothing has been stopped yet. Having a variable called `*_killed` or `*_stopped` implies those things have already been `killed`/`stopped`, this is not the case here.

A name like `jobs_to_die` would make it clear that the jobs are still alive at line 304.

Reply:
Again, the list is meant to store jobs that were stopped. 304 is the initialisation point, but the meaning of a variable is not in general given by the initialisation point. I'll change `killed` to `stopped`.